test/e2e/specs/plans/plans__signup-business.ts

```diff
@@ -73,14 +73,18 @@ describe(
 			} );
 
 			it( 'Skip Onboarding', async function () {
+				await page.waitForURL( /setup\/site-setup\/goals/, { waitUntil: 'networkidle' } );
 				const startSiteFlow = new StartSiteFlow( page );
-				await startSiteFlow.clickButton( 'Skip to dashboard' );
+				await Promise.all( [
+					page.waitForURL( /build\/launchpad/, { waitUntil: 'networkidle' } ),
```

Comment:
`NB:` Is there a reason we chose `networkidle` instead of `load`?

```
'networkidle' - consider operation to be finished when there are no network connections for at least 500 ms.

'load' - consider operation to be finished when the load event is fired.
```

Reply:
@jeyip Yes, the `networkidle` is for added safety. 

The default `waitForURL` call without the added parameter has an implicit wait for `load` state, which is good enough most of the time. I will remove the `networkidle` and we can observe whether it reduces/eliminates incidents of flakiness.